migrations/v3.js

```diff
@@ -0,0 +1,79 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin, getCourse, testStopWhere, testSuccessWhere } from 'adapt-migrations';
+import _ from 'lodash';
+
+describe('Resources - v2.1.3 to v3.0.0', async () => {
+  let course, courseResourcesGlobals;
+  const originalDescriptionDefault = 'Click here to view resources for this course';
+  whereFromPlugin('Resources - from v2.1.3', { name: 'adapt-contrib-resources', version: '<3.0.0' });
+
+  whereContent('Resources - where Resources', async (content) => {
+    course = getCourse();
+    return course?._resources;
+  });
+
+  mutateContent('Resources - modify description default', async (content) => {
+    if (course._resources.description === originalDescriptionDefault) course._resources.description = 'Select here to view resources for this course';
+    return true;
+  });
+
+  mutateContent('Resources - add globals if missing', async (content) => {
+    if (!_.has(course, '_globals._extensions._resources')) _.set(course, '_globals._extensions._resources', {});
+    courseResourcesGlobals = course._globals._extensions._resources;
+    return true;
+  });
+
+  mutateContent('Resources - replace globals resourcesEnd with resources', async (content) => {
+    if (_.has(courseResourcesGlobals, 'resourcesEnd')) {
+      courseResourcesGlobals.resources = courseResourcesGlobals.resourcesEnd;
```

Comment:
Since the purpose of this property has completely [changed](https://github.com/adaptlearning/adapt-contrib-resources/commit/72cc8e16096dcd2dd88bccbd7bd35b6ff06f3adc), should we just delete `resourcesEnd` and always use the new default value for `resources`, "Additional resources"? The previous value for `resourcesEnd` let the user know they are at the _end_ of the list, while the renamed `resources` property is meant to _introduce_ the list.